integrasjonspunkt/src/test/java/no/difi/meldingsutveksling/noarkexchange/IntegrasjonspunktImplTest.java

```diff
@@ -22,6 +24,10 @@
 
     @Mock
     private InternalQueue queueMock;
+    @InjectMocks
+    private EDUCoreService coreServiceMock;
+    @InjectMocks
```

Comment:
Bør unngå å bruke @ InjectMocks . Artikkel under forklarer hvorfor. Det finnes mange andre som skriver det samme (kanskje tom. også Mockito sin dokumentasjon).
https://tedvinke.wordpress.com/2014/02/13/mockito-why-you-should-not-use-injectmocks-annotation-to-autowire-fields/